.github/workflows/infra_plan.yaml

```diff
@@ -9,6 +9,11 @@ on:
         description: Azure region where the resources will be deployed.
         type: string
         required: true
+      tf_configuration_path:
+        description:
+        type: string
+        required: false
+        default: infra/resources
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** I'm personally more keen on a different design:
* don't assume default to `infra/resources`: it adds no much sugar while hiding a useful piece of information (considering that we plan to move this workflow to another shared repo)
  * hence make the parameter required 
* shorter property name (but that's really a nitpick)
‏

```suggestion
      base_path:
        description: The base path on which the script will look for Terraform projects
        type: string
        required: true
```

Reply:
Good feedback, I agree on naming since yours is better. Shall we also add `tf` as prefix?

I am not sure about the default value instead. We are also using `region` and `environment` variables to build the path, and probably the value for `base_path` is not that clear from a consumer point of view. Using a default value, we are limiting scenarios to the exception cases only

Reply:
That's ok, I got rid of them